ToNote/Views/BaseDialogWindow.xaml

```diff
@@ -7,6 +7,27 @@
         SizeToContent="WidthAndHeight"
         Title="{Binding Title}" Height="200" Width="500"
         WindowStartupLocation="CenterScreen"
+        WindowStyle="None"
         >
-    <ContentControl x:Name="ContentPresenter" Content="{Binding}"/>
+    <WindowChrome.WindowChrome>
+        <WindowChrome CaptionHeight="40" />
+    </WindowChrome.WindowChrome>
+    <Grid Background="Transparent">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="40" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+            <DockPanel VerticalAlignment="Center" Grid.Row="0">
+                <TextBox Text="ToNote" Height="30"  HorizontalAlignment="Left" BorderThickness="0" Background="Transparent" Foreground="#eeeeee"/>
+            <StackPanel WindowChrome.IsHitTestVisibleInChrome="True" Orientation="Horizontal" HorizontalAlignment="Right" >
+                <Button Click="Close_Window"  Style="{StaticResource EmptyButtonStyle}">
+                        <Button.Content>
+                            <Image Source="https://i.imgur.com/I11UYRS.png?1" />
```

Comment:
Getting images from links is a bad idea. Consult @t-eliks on how to implement this the right way.

Reply:
We decided since I will be implementing the WindowStyle buttons as Paths, that he should just insert some icons from online for demo and testing purposes.